src/main/java/net/imagej/omero/DefaultOMEROService.java

```diff
@@ -410,7 +410,7 @@ public long uploadTable(final OMEROCredentials credentials,
 				}
 			}
 			finally {
-				session.close();
+				((DefaultOMEROSession) session).close();
```

Comment:
Also, now that we have Java 7 features available, you can use try-with-resources to manage the session, and you won't have to call `close()` manually in the finally block regardless.